src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -260,7 +255,7 @@ public function setLoggers(array $loggers)
      *
      * @return callable|null The previous exception handler
      */
-    public function setExceptionHandler(callable $handler = null)
+    public function setExceptionHandler(callable $handler = null): ?callable
```

Comment:
`->setExceptionHandler()` calling to a setter method without arguments look bad, now that we can we should change it to `?callable $handler` (removing the default value). 

If anyone wants to set it to null, they must explicitly pass it on: `->setExceptionHandler(null)`.